src/rust/ide/lib/view/src/graph_editor.rs

```diff
@@ -894,9 +896,9 @@ impl Deref for GraphEditorModelWithNetwork {
 }
 
 impl GraphEditorModelWithNetwork {
-    pub fn new<S:Into<Scene>>(scene:S, cursor:cursor::Cursor) -> Self {
+    pub fn new(world:&World, cursor:cursor::Cursor) -> Self {
```

Comment:
Please, do not do this change. Scene was here on purpose. Passing world is way too generic.

Reply:
@wdanilo `ProjectName` uses `TextField` which needs `World` to be created. This can be refactored out when we get the new `TextField` implementation. Sounds good?

Reply:
Why cant we change the impl of old text field to get `Into<Scence>` instead?

Reply:
@wdanilo because the old `TextField` needs to get `FocusManager` from `World`.

Reply:
I changed the implementation of TextField to get `Into<Scene>`. 1h later :)